resources/metadata/npmExecuteScripts.yaml

```diff
@@ -22,6 +22,15 @@ metadata:
     - pattern: "ERR_PNPM_FETCH_401"
       message: "PNPM authentication failed. Check your credentials or token."
       category: "authentication"
+    - pattern: "npm error ERESOLVE"
+      message: "NPM dependency resolution failed. Review peer dependency conflicts."
+      category: "dependency"
+    - pattern: "npm error EINTEGRITY"
+      message: "Package integrity check failed. Clear npm cache and retry installation."
+      category: "dependency"
+    - pattern: "npm error code ENOENT.*package.json"
+      message: "Package.json file not found. Ensure package.json exists in the correct directory."
```

Comment:
Would be good to say to check if `buildDescriptorList` and `buildDescriptorExcludeList` are correct.